### DISCUSSION TITLE
Is there a gatsby plugin for prisma?

### DISCUSSION THREAD

**Discussion by theprantadutta** [prisma/prisma#4620] [Q&A] [answered]
Hi, So I just want to know is there a gatsby plugin for prisma? because I can't find one in gatsby plugins library. If there is not, then How would I use gatsby with prisma?

**Comment by nikolasburk** [comment, marked as answer]
Hey @prantaDutta, I don't think there's a Gatsby plugin for Prisma yet and I personally haven't used the two together. It seems like this repo might be a good starting point to explore how the two fit together: https://github.com/LekoArts/gatsby-with-prisma-client

It also depends a bit on what exactly your use case is for using Prisma with Gatsby. If your Gatsby frontend is talking to an API server (e.g. a REST API written with [Express](https://www.prisma.io/express) or a GraphQL API with [Apollo Server](https://www.prisma.io/apollo)), Prisma is used as an ORM in the API server just like any other ORM like TypeORM or Sequelize). 

If you want to use it to build static sites, you can query the database at build time using Prisma. Also, if building static sites is your use case you might want to check out [Next.js](https://nextjs.org/) which is a great alternative to Gatsby and [integrates perfectly with Prisma](https://www.prisma.io/nextjs).

**Reply by theprantadutta** [reply]
Thanks, @nikolasburk. I am using nextjs right now with Prisma. It's just I thought maybe gatsby is a bit faster since it has graphql already built-in.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
